libexec/basher-upgrade

```diff
@@ -14,6 +14,11 @@ if [ "$1" == "--complete" ]; then
   exec basher-list
 fi
 
+if [ "$1" == "--all" ]; then
+  basher-list | xargs -n1 basher upgrade
+  exit 0
```

Comment:
Should this be `exit`, so if any of the `basher upgrade`s fail, the non-zero exit code is perserved?

Reply:
Fair point. I've adapted it!